ui/documents/templates/semantic-ui/documents/Sidebar.jinja

```diff
@@ -1,46 +1,47 @@
 {#def metadata, record, extra_context, is_preview, api_record #}
-{%set recordRequests = record.expanded.requests and record.expanded.requests|selectattr('status_code', 'in', ['submitted', 'created'])|list%}
+{%set recordRequests = record.expanded.requests and record.expanded.requests|selectattr('status_code', 'in',
+['submitted', 'created'])|list%}
 {% if record.expanded.request_types or recordRequests or record.links.edit_html %}
 <section class="ui segment" aria-label='{{ _("Actions") }}'>
-<h2 class="ui small header detail-sidebar-header">
-  {{ _("Actions") }}
-  <a href="{{record.links.requests_html}}" target="_blank" rel="noopener noreferrer" title="{{_("Search record's requests")}}">
-    <i class="eye icon black"></i>
-  </a>
-</h2>
-{% if record.links.edit_html%}
+  <h2 class="ui small header detail-sidebar-header">
```

Comment:
Ideally, the whole following markup should be a react widget...Doing so will enable other repo developers to reuse it without copying html blocks here from documents...as it is generic functionality bound to deposit form, I think it should live in oarepo-ui